cli/script/command-executor.ts

```diff
@@ -742,8 +742,8 @@ function promote(command: cli.IPromoteCommand): Promise<void> {
 }
 
 function release(command: cli.IReleaseCommand): Promise<void> {
-    if (command.package.search(/\.zip$/i) !== -1) {
-        throw new Error("It is unnecessary to package releases in a .zip file. Please specify the path of the desired directory or file directly.");
+    if (isBinaryOrZip(command.package)) {
+        throw new Error("It is unnecessary to package releases in a .zip or binary file. Please specify the direct path to the update content's directory (e.g. /platoforms/ios/www) or file (e.g. main.jsbundle).");
```

Comment:
Sorry, I had a typo in my string suggestion :) It should be `/platforms/ios/www'.